app/views/user_mailer/prompter_notification.html.erb

```diff
@@ -1,29 +1,30 @@
 <% content_for :message do %>
   <p>
-    A response to your prompt has been posted <% if @collection %>in the <%= style_link(@collection.title, collection_url(@collection)) %> collection <% end %>at the Archive of Our Own!
+    <% if @collection %>
+      <%= t(".a_response_to_your_prompt.collection.html",
+            collection_link: style_link(@collection.title, collection_url(@collection))) %>
+    <% else %>
+      <%= t(".a_response_to_your_prompt.text") %>
+    <% end %>
   </p>
 
   <p>
-    <% if @collection.nil? %>
-      <%= style_creation_link(@work.title, work_url(@work)) %> (<%= @work.word_count %> words)
-    <% else %>
-      <%= style_creation_link(@work.title, collection_work_url(@collection, @work)) %> (<%= @work.word_count %> words)
-    <% end %>
+    <% url = @collection ? collection_work_url(@collection, @work) : work_url(@work) %>
+    <%= creation_link_with_word_count(@work, url) %>
     <br>
-    by <%= @work.anonymous? ? style_bold("an anonymous responder") : (@work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe) %>
+    <%= t(".byline.html", work_creator_links: creator_links(@work)) %>
     <br>
-    <% unless @work.fandom_string.blank? %>
-      <%= style_bold("Fandom:") %> <%= @work.fandom_string %>
+    <% if @work.fandom_string.present? %>
+      <%= style_bold(t(".label.fandom")) %> <%= @work.fandom_string %>
     <% end %>
   </p>
-  
-  <% unless @work.summary.blank? %>
-    <%= style_bold("Summary:") %>
+
+  <% if @work.summary.present? %>
+    <%= style_bold(t(".label.summary")) %>
     <%= style_quote(raw sanitize_field(@work, :summary)) %>
```

Comment:
I don't know if we'd want to change it in this pull request, or make an issue to change it in all the emails at once, but we should modify this to use the work_info partial, which contains more information and lets us reuse translations/code.

Reply:
Just following up to say it looks like we already have one for this email! [AO3-5764](https://otwarchive.atlassian.net/browse/AO3-5764)